src/snyk/snykCode/error/snykCodeErrorHandler.ts

```diff
@@ -34,6 +36,10 @@ export class SnykCodeErrorHandler extends ErrorHandler implements ISnykCodeError
     this.transientErrors = 0;
   }
 
+  get transientErrorsLimitExhausted(): boolean {
+    return this.transientErrorLimitHit;
```

Comment:
Why the difference in naming?

Reply:
just that `transientErrorsLimitExhausted` is a `public get` method on that class which reflects the value of a private variable `transientErrorLimitHit` which further records the `boolean` state of whether the limit was hit.

Reply:
renamed the `transientErrorLimit` to `_transientErrorsLimitExhausted` to keep things consistent

Reply:
renamed the `transientErrorLimit` to `_transientErrorsLimitExhausted` to keep things consistent